client/src/components/glossary_components.js

```diff
@@ -0,0 +1,50 @@
+import { IconQuestion } from '../icons/icons.js';
+import { GlossaryEntry } from '../models/glossary.js';
+import { glossary_href } from '../link_utils.js';
+
+const GlossaryTooltipItem = ({id, arrow_selector, inner_selector, children}) => (
+  <a
+    href={( window.is_a11y_mode || window.feature_detection.is_mobile() ) ? glossary_href(id) : null}
```

Comment:
Were the tooltips not working on mobile then?

Reply:
Oops, should have done this as a review. This is my one question, approve the rest!

Reply:
They do, this was wrong!